src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -1,5 +1,17 @@
 var NEW_QUESTION = -1;
 
+var WARNING_CANCEL = 'Warning: Any unsaved changes will be lost';
+var CONFIRM_CANCEL_EDIT = 'Are you sure you want to cancel your edit?';
+var CONFIRM_CANCEL_ADD_QNS = 'Are you sure you want to cancel adding this question?';
+
+var WARNING_DELETE_QNS = 'Warning: Deleted question cannot be recovered';
+var CONFIRM_DELETE_QNS = 'Are you sure you want to delete this question?';
+
+var WARNING_DELETE_RESPONSES = 'Warning: Existing responses will be deleted by your action';
+var CONFIRM_DELETE_RESPONSES =
```

Comment:
Might have to change the name of these two variables to something like `*_EDIT_DELETES_RESPONSES` (or `*_EDIT_REMOVES_RESPONSES` if the "delete" makes it confusing), as on first glance this looks like the result of a delete action.